mrmustard/lab_dev/states/ket.py

```diff
@@ -273,6 +273,14 @@ def expectation(self, operator: CircuitComponent):
 
         return result
 
+    def fidelity(self, other: State) -> float:
+        r"""
+        The fidelity between this ket and another state.
+        """
```

Comment:
Seems like the fidelity is defined with a squared. I suggest clarifying it i.e., adding

```suggestion
        r"""
        The fidelity between this ket and another state.
        
        .. details::
        
        .. math::
            F(|\psi\rangle, \phi\rangle) = |\langle \psi, \phi \rangle|^2
        """
```